src/Elasticsearch/Serializer/ItemNormalizer.php

```diff
@@ -111,7 +111,7 @@ public function getSupportedTypes($format): array
             ];
         }
 
-        return DocumentNormalizer::FORMAT === $format ? $this->decorated->getSupportedTypes($format) : [];
+        return DocumentNormalizer::FORMAT !== $format ? $this->decorated->getSupportedTypes($format) : [];
```

Comment:
not sure about: 

```suggestion
        return DocumentNormalizer::FORMAT === $format ? $this->decorated->getSupportedTypes('json') : [];
```

@dunglas any idea?